index.bs

```diff
@@ -10877,35 +10877,74 @@ defined on that interface.
+            given |attr|, |interface|,  and |realm|.
+        1.  Let |setter| be the result of creating an [=attribute setter=]
+            given |attr|, |interface|,  and |realm|.
+
+            Note: the algorithm to create an [=attribute setter=]
+            returns <emu-val>undefined</emu-val> if |attr| is [=read only=].
+        1.  Let |desc| be the PropertyDescriptor{\[[Get]]: |getter|, \[[Set]]: |setter|,
+            \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: <emu-val>true</emu-val>}.
+        1.  Let |id| be |attr|'s [=identifier=].
+        1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|interfaceProtoObj|, |id|, |desc|).
+        1.  If |attr| is declared with the [{{Unscopable}}] [=extended attribute=], then
+            perform [=!=] <a abstract-op>CreateDataProperty</a>(|unscopableObject|, |id|,
+            <emu-val>true</emu-val>).
+    1.  For each [=exposed=] [=regular operation=] |op| that is a [=member=] of |interface|,
+        1.  Let |method| be the result of [=creating an operation function|creating an operation function=]
+            given |op|, |interface|, and |realm|.
+        1.  Let |id| be |op|'s [=identifier=].
+        1.  Perform [=!=] <a abstract-op>CreateDataProperty</a>(|interfaceProtoObj|, |id|, |method|).
+        1.  If |op| is declared with the [{{Unscopable}}] [=extended attribute=], then
+            perform [=!=] <a abstract-op>CreateDataProperty</a>(|unscopableObject|, |id|, <emu-val>true</emu-val>).
+    1.  For each [=exposed=] [=constant=] |const| that is a [=member=] of |interface|,
+        1.  Let |value| be the result of [=converted to an ECMAScript value|converting=]
+            |const|’s IDL value to an ECMAScript value.
+        1.  Let |desc| be the PropertyDescriptor{\[[Writable]]: <emu-val>false</emu-val>,
+            \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: <emu-val>false</emu-val>,
+            \[[Value]]: |value|}.
+        1.  Let |id| be |const|'s [=identifier=].
+        1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|interfaceProtoObj|, |id|, |desc|).
+    1.  If the [{{NoInterfaceObject}}] [=extended attribute=] was not specified on |interface|, then:
+        1.  Let |constructor| be the [=interface object=] of |interface|.
+        1.  Let |desc| be the PropertyDescriptor{\[[Writable]]: <emu-val>true</emu-val>,
+            \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>true</emu-val>,
+            \[[Value]]: |constructor|}.
+        1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|interfaceProtoObj|, "<code>constructor</code>", |desc|).
+    1.  Let |value| be the concatenation of the |interface|’s [=identifier=]
+        and the string "<code>Prototype</code>".
+    1.  Let |desc| be the PropertyDescriptor{\[[Writable]]: <emu-val>false</emu-val>,
+        \[[Enumerable]]: <emu-val>false</emu-val>, \[[Configurable]]: <emu-val>false</emu-val>,
+        \[[Value]]: |value|}.
```

Comment:
https://heycam.github.io/webidl/#issue-48167bcc says:

> Should define whether `@@toStringTag` is writable, enumerable and configurable. All `@@toStringTag` properties in the ECMAScript spec are non-writable and non-enumerable, and configurable.

I assume this "issue" has been resolved?